src/v3/src/components/AuthenticatorButton/AuthenticatorButton.tsx

```diff
@@ -100,6 +104,79 @@ const AuthenticatorButton: UISchemaElementComponent<{
     return <ArrowRightIcon titleAccess={ctaLabel} />;
   }
 
+  function renderDescription() {
```

Comment:
Prefer arrow functions here

Reply:
ah yeah was just following the style in the above method, updated both to use arrow functions